Tests/ProtocolBufferConversionTests.swift

```diff
@@ -448,6 +448,9 @@ final class ProtocolBufferConversionTests: XCTestCase {
 
   func testConvertPaymentTransactionAllCommonFieldsSet() {
     // GIVEN a Transaction protocol buffer with all common fields set.
+    let hash = Data([2, 4, 6])
+    let timestamp: UInt32 = 0
+    let deliveredAmount: UInt64 = 20
```

Comment:
Are these the 3 fields that were added to Transactions in TransactionHistory?

Reply:
I'm not sure I totally understand the question... but these 3 fields are fields that are present in the XpringWalletSystem's notion of a transaction.  Therefore we needed to plumb them through in the SDK in order for the SDK to be sufficient as an underlying messenger for `transactionHistoryForWallet` of the `XpringWalletSystemClient`.

Reply:
Are these fields not part of a `Transaction` object that you get from `rippled`?

Reply:
These three fields are part of a `GetTransactionResponse` protobuf object that we get from `rippled` via `gRPC`